### PR TITLE
[DGPFUS-162] Create & Style the Landing Page for a Food Orderin

### PR DESCRIPTION
**Pull request by Lisw4niso**
CSS task 1 implementation

### CODE REVIEW

File: submissions/liswanisoLubinda/css/html/index.html [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+                <p>Anything food, we have you covered.</p>
+            </div>
+        </section>
+
+        <!--Section and div to create form for customer to select their addresss-->
+        <section class="address">
+            <div class="customer_address">
+                <h3>Where are you?</h3>
+                <form action="#" method="post" enctype="multipart/form-data">
+                    <select name="address">
```
Comment (Mayowa2020) [inline submissions/liswanisoLubinda/css/html/index.html:35]: in place of the select tag, please use datalist with an input field and a placeholder.

**Review by Mayowa2020 — CHANGES_REQUESTED** [review]
Hi @Lisw4niso Try to run the test on the code and in case you experience any blocker, please do not hestitate to reach out to me. Thank you,

File: submissions/liswanisoLubinda/css/html/index.html (hunk 1 above)
Comment (Lisw4niso) [reply submissions/liswanisoLubinda/css/html/index.html:35]: Noted.

**Review by Mayowa2020 — APPROVED** [review]
Well done @Lisw4niso! Approved...

**Pull request merged**